presto-docs/src/main/sphinx/kerberos/hive.rst

```diff
@@ -0,0 +1,162 @@
+=====================
+Hive Kerberos Support
+=====================
+
+The :doc:`/connector/hive` supports connecting to an instance of Hive running on
+a Hadoop cluster that has Kerberos authentication enabled. There are two
+services on the Hadoop cluster that can be configured to require Kerberos
+authentication:
+
+* The Hive metastore Thrift service
+* The Hadoop Distributed File System (HDFS)
+
+Access to these services by the Hive connector is configured in the properties
+file that contains the general Hive connector configuration.
+
+Hive Metastore Thrift Service
+-----------------------------
+
+In a Kerberized Hadoop cluster, access to the Hive metastore Thrift service is
+done using the Simple Authentication and Security Layer (SASL). Kerberos
+authentication for the metastore is configured in the connector's properties
+file using the following properties:
+
+================================================== ============================================================ ==========
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ==========
+``hive.metastore.sasl.enabled``                    When set to ``true`` the Hive connector will connect to the  ``false``
+                                                   Hive metastore Thrift service using SASL and authenticate
+                                                   with Kerberos. When using the default value of ``false``,
+                                                   no Kerberos authentication is disabled and no other
```

Comment:
I think it would be better to just delete the first "no" in this line so you have:
When using the default value of ``false``, Kerberos authentication is disabled...